packages/contracts/src/rbac-client.ts

```diff
@@ -0,0 +1,144 @@
+import { Client, Provider, Receipt } from "@blockstack/clarity"
+import { TextEncoder } from "util";
+import { Clarity } from "./util"
+
+
+export class RBACClient extends Client {
+    constructor(principal: string, provider: Provider) {
+        super(
+            `${principal}.rbac`,
+            'rbac',
+            provider
+          );
+    }
+
+    async getRoles(user: string): Promise<Receipt> {
+      const query = this.createQuery({
+        method: {
+          name: 'get-roles',
+          args: [
+            Clarity.principal(user)
+          ]
+        }
+      });
+
+      const receipt = await this.submitQuery(query);
+      return receipt;
```

Comment:
I thought about it and decided to postpone the decision about if and how we could simplify query and transaction creation until we have more code.

It's tempting to do it right now, but we may find a better way or maybe we decide to create PR with such change to official library, so that others could benefit from it as well.